codex-ui/src/vue/components/alert/BaseAlert.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div
+    ref="el"
+  >
```

Comment:
you can specify a theme using `theme-base` attribute. For example for "success":

```html
<div theme-base"grass" />
```